enctests/test_configs/color_tests.yml

```
@@ -0,0 +1,174 @@
+
+---
+test_colormatch_raw:
+  app: ffmpeg
+  comparisontest:
+  - testtemplate: idiff {originalfile} {newfile}
+    testtype: idiff
+  - testtype: assertresults
+    tests:
+    - assert: less
+      value: max_error
+      less: 0.00195
+  description: Test from /Users/sam/git/EncodingGuidelines/enctests/../ColorPreservation.md
+    line 44
+  encoding_template: ffmpeg -y {input_args} -i "{source}"                 {encoding_args}
+    -y "{outfile}"
+  name: test_colormatch_raw
+  sources:
+  - /Users/sam/git/EncodingGuidelines/sourceimages/chip-chart-1080-noicc.png.yml
```

```suggestion
    - /Users/sam/git/EncodingGuidelines/sourceimages/chip-chart-1080-noicc.png.yml
```
I think this is missing an indent?

---

It does seem to work, you can see further up where I'm doing the same with the tests.